src/zeit/content/volume/volume.py

```diff
@@ -120,3 +120,14 @@ def __getattr__(self, key):
     def __setattr__(self, key, value):
         """Interfere with zope.formlib and store content via setitem."""
         self[key] = value
+
+
+@grok.adapter(zeit.cms.content.interfaces.ICommonMetadata)
+@grok.implementer(zeit.content.volume.interfaces.IVolume)
+def retrieve_volume_using_info_from_metadata(context):
+    if context.year is None or context.volume is None:
+        return None
+    uniqueId = 'http://xml.zeit.de/ausgabe/{year}/{name}'.format(
```

Comment:
Machen wir "konfiguriere das Namensschema in der product.xml" jetzt noch oder später oder wie?